core/src/main/java/io/seata/core/rpc/netty/NettyClientConfig.java

```diff
@@ -52,6 +52,7 @@
     private static final boolean DEFAULT_POOL_TEST_BORROW = true;
     private static final boolean DEFAULT_POOL_TEST_RETURN = true;
     private static final boolean DEFAULT_POOL_LIFO = true;
+    private static final boolean ENABLE_CLIENT_BATCH_SEND_REQUEST = CONFIG.getBoolean("transport.enable-client-batch-send-request", true);
```

Comment:
use constants

Reply:
> use constants

I will create a new pr to change the netty part constant